parcels/scripts/check_corrupted_files.py

```diff
@@ -0,0 +1,35 @@
+"""
+Function to check if the data of a variable in a field list is corrupted
+"""
+
+import numpy as np
+import xarray as xr
+import os
+
+
+def check_corrupted_files(filelist, fieldname, outputfile=None):
+    """
+    parameter filelist: list of files which contain the field with name 'fieldname'
+    parameter fieldname: name of the field that is checked
+    parameter outputfile: list of corrupted files is written into a output text file
+    """
+
+    if outputfile is None:
+        outputfile = 'CorruptedFiles_field_' + fieldname + '.txt'
+
+    if os.path.isfile(outputfile):
+        print 'Outputfile exists already. Please delete or choose another name for the output file.'
```

Comment:
Write all print statements in python3 format

Reply:
Done!